sdk/core/core-http/src/webResource.ts

```diff
@@ -438,7 +438,7 @@ export class WebResource implements WebResourceLike {
       this.headers.set("Content-Type", "application/json; charset=utf-8");
     }
 
-    // set the request body. request.js automatically sets the Content-Length request header, so we need not set it explicilty
+    // set the request body. request.js automatically sets the Content-Length request header, so we need not set it explicitly
```

Comment:
Nice to fix the typo but feels out of place in this SB pr?

Reply:
Hmmm, I'll revert.

Reply:
```suggestion
    // set the request body. request.js automatically sets the Content-Length request header, so we need not set it explicilty
```
Reverting the typo fix since it is of core-http and out of scope for this PR